guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -59,7 +59,7 @@ endif::[]
 // Upgrading Capsule Server
 include::topics/upgrading_capsule_server.adoc[leveloffset=+2]
 
-ifdef::katello,orcharhino,satellite[]
+ifndef::satellite[]
```

Comment:
```suggestion
ifdef::katello,orcharhino[]
```

Sorry, I didn't see it was already conditioned (and you did it the right way around in the other places)